segmentation_models_pytorch/encoders/_efficientnet.py

```diff
@@ -171,12 +171,11 @@ def set_swish(self, memory_efficient=True):
         Args:
             memory_efficient (bool): Whether to use memory-efficient version of swish.
         """
-        self._swish = MemoryEfficientSwish() if memory_efficient else Swish()
+        self._swish = MemoryEfficientSwish() if memory_efficient else nn.SiLU()
```

Comment:
I suppose Pytorch implementation is memory efficient, so we can just get rid of custom `MemoryEfficientSwish` class